___________________________________ NUEVO ERROR ___________________________________
Error en el men√∫ principal.
2020-06-04 19:23:14

[sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method), sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62), sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45), java.lang.reflect.Constructor.newInstance(Constructor.java:423), com.mysql.jdbc.Util.handleNewInstance(Util.java:425), com.mysql.jdbc.Util.getInstance(Util.java:408), com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944), com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933), com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869), com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524), com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675), com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465), com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1912), com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2020), proyecto.controlador.ControladorEjercicio.obtenerCodigosExistentes(ControladorEjercicio.java:178), proyecto.controlador.ControladorEjercicio.imprimirCodigosExistentes(ControladorEjercicio.java:156), proyecto.controlador.ControladorEntrenamiento.crearLineaEntrenamiento(ControladorEntrenamiento.java:382), proyecto.controlador.ControladorEntrenamiento.crearNuevoEntrenamiento(ControladorEntrenamiento.java:43), proyecto.vista.MenuPrincipal.menuCrearEntrenamiento(MenuPrincipal.java:155), proyecto.vista.MenuPrincipal.menuEntrenador(MenuPrincipal.java:108), proyecto.vista.MenuPrincipal.main(MenuPrincipal.java:50)]

__________________________________________________________________________________

